types/chrome-apps/index.d.ts

```diff
@@ -11632,7 +11632,7 @@ declare class HTMLWebViewElement extends HTMLElement {
      * If the storage partition ID starts with persist: (partition='persist:googlepluswidgets'),
      * the webview will use a persistent storage partition available to all guests in the app with the same storage partition ID.
      * If the ID is unset or if there is no 'persist': prefix, the webview will use an in-memory storage partition.
-     * his value can only be modified before the first navigation, since the storage partition of an active renderer process cannot change.
+     * This value can only be modified before the first navigation, since the storage partition of an active renderer process cannot change.
```

Comment:
The documentation for the Chrome APIs is copied verbatim from the official documentation pages written by Google. I am hesistant to changing that. If Google changes their documentation to use gender-neutral pronouns, those changes will propagate down to these type definitions.

Reply:
The docs seem to already have the typo fix proposed: https://developer.chrome.com/apps/tags/webview#partition